liatrio/builder-images
# ENG-1420 - Add Gradle and Ruby Builder Images

This PR adds Gradle and Ruby languages to the builder-images list to allow support in Jenkins/CI for these languages/build tools.

### Changes
- Added Gradle & Ruby builder images
- Updated README.md to include all current build images
- Renamed `images` directory to `img`

### Testing
- Check that Jenkins status passes
- Checkout branch locally and run `skaffold build` to ensure nothing was broken
